src/aggregationserver/experimenttracker.cpp

```diff
@@ -47,6 +47,9 @@ int ExperimentTracker::RegisterExperimentRun(
             experiment_name
         };
         experiment_id = database_->InsertValuesUnique("Experiment", columns, values, {"Name"});
+        if (experiment_id == -1) {
+            throw std::runtime_error("Unable to register experiment");
```

Comment:
Give more context as to which experiment run registration failed. eg experiment id or name.

Reply:
Addressed in [this commit](https://github.com/cdit-ma/logan/pull/40/commits/dfe7a996577b46a65738668510cc377870085be8)